### PR TITLE
fix(workbook): fix abs path  /xl/worksheets/sheet1.xml -> xl/worksheets/sheet1.xml

### PR DESCRIPTION
**Pull request by stolen-programmer**
fix abs path  /xl/worksheets/sheet1.xml -> xl/worksheets/sheet1.xml

### CODE REVIEW

File: QXlsx/source/xlsxworkbook.cpp [hunk 1]
```diff
@@ -615,6 +615,11 @@ bool Workbook::loadFromXmlFile(QIODevice *device)
                 QString fullPath =
                     QDir::cleanPath(parts.first() + QLatin1String("/") + relationship.target);
 
+                // fix abs path  /xl/worksheets/sheet1.xml -> xl/worksheets/sheet1.xml
+                if (relationship.target.startsWith(QString::fromUtf8("/"))) {
```
Comment (dantti) [inline QXlsx/source/xlsxworkbook.cpp:619]: please replace costy fromUtf8 with u'/' and can you explain why this change is needed?

Comment (stolen-programmer) [reply QXlsx/source/xlsxworkbook.cpp:619]: I am unable to read an xlsx
Through breakpoint localization
Confirmed that files starting with/cannot be read

File: QXlsx/source/xlsxworkbook.cpp [hunk 2]
```diff
@@ -616,7 +616,7 @@ bool Workbook::loadFromXmlFile(QIODevice *device)
                     QDir::cleanPath(parts.first() + QLatin1String("/") + relationship.target);
 
                 // fix abs path  /xl/worksheets/sheet1.xml -> xl/worksheets/sheet1.xml
-                if (relationship.target.startsWith(QString::fromUtf8("/"))) {
+                if (relationship.target.startsWith(u"/")) {
```
Comment (dantti) [inline QXlsx/source/xlsxworkbook.cpp:619]: it's a single char use u'/'

File: QXlsx/source/xlsxworkbook.cpp (hunk 1 above)
Comment (stolen-programmer) [reply QXlsx/source/xlsxworkbook.cpp]: You can take a look at the details
#305

File: QXlsx/header/xlsxworksheet.h [hunk 3]
```diff
@@ -192,6 +192,9 @@ class QXLSX_EXPORT Worksheet : public AbstractSheet
 private:
     void saveToXmlFile(QIODevice *device) const override;
     bool loadFromXmlFile(QIODevice *device) override;
+
+public:
+    const QList<CellRange> &editRegion() const;
```
Comment (dantti) [inline QXlsx/header/xlsxworksheet.h:197]: Please change to:
QList<CellRange> editRegion() const;
as const & return in Qt containers is not needed

File: QXlsx/source/xlsxworksheet.cpp [hunk 4]
```diff
@@ -2444,6 +2443,48 @@ void WorksheetPrivate::loadXmlSheetData(QXmlStreamReader &reader)
...
+                if (editRegions.empty()) {
+                    editRegions << CellRange{pos, pos};
+                } else {
+                    auto newRegion = false;
+                    auto cellRange = editRegions.last();
+
+                    // 跨多列
+                    if (pos.column() - cellRange.firstColumn() < -1 ||
+                        pos.column() - cellRange.lastColumn() > 1 || // 跨多列
+                        // 跨多行
```
Comment (dantti) [inline QXlsx/source/xlsxworksheet.cpp:2460]: Please leave comments in english.

File: QXlsx/header/xlsxworksheet.h (hunk 3 above)
Comment (dantti) [reply QXlsx/header/xlsxworksheet.h:197]: also since this is plublic plese move upwards and leave a description documenting it

**Review by dantti — CHANGES_REQUESTED** [review]

**Pull request closed without merging**